app/src/routes/create.js

```diff
@@ -0,0 +1,10 @@
+import { Link } from "react-router-dom";
+
+export default function Create() {
```

Comment:
Could we rename this to "Upload", and then create a route to another page called "Configure". It better fits the mockups that way

Reply:
Should the upload and configure (same goes for the printer side) be 2 separate pages entirely? Is a react page like a full on new html page, where when you click on a new route you load the selected page? If that is the case then shouldn't it be that the upload and configure are elements of one page that you can click between? To allow seamless transition and less loading.

Reply:
Yes I suppose we could do that. I believe you're right that a new route demands loading a new page. I was just thinking about organization - it's possible that we'll add more steps to the create jobs process, so the question is how much code are we willing to support in a single page?

Reply:
The idea I had here is that each form stage shouldn't be a new route since it wouldn't make sense for someone to navigate directly to the second stage (which would be possible on separate routes). Further, the two stages would need to share state so keeping it on the same route simplifies it. Instead of two routes, the one route would use conditional rendering to flip between the "Upload" and "Configure" sections of the form.